ios/Classes/TiMapView.m

```diff
@@ -1128,6 +1143,20 @@ -(void)handleLongPressOnMap:(UIGestureRecognizer *)sender
     }
 }
 
+-(void)handleMapClick:(MKMapPoint)point
+{
+	TiProxy * mapProxy = [self proxy];
+
+    CLLocationCoordinate2D clickCoordinate = MKCoordinateForMapPoint(point);
+    NSNumber *lat = [NSNumber numberWithDouble:clickCoordinate.latitude];
+    NSNumber *lng = [NSNumber numberWithDouble:clickCoordinate.longitude];
+    NSDictionary * event = [NSDictionary dictionaryWithObjectsAndKeys:
+                            mapProxy,@"map", lat,@"latitude", lng,@"longitude", nil];
+    if ([mapProxy _hasListeners:@"mapclick"]) {
+        [mapProxy fireEvent:@"mapclick" withObject:event];
+    }
+}
```

Comment:
<img width="714" alt="bildschirmfoto 2017-03-06 um 11 49 33" src="https://cloud.githubusercontent.com/assets/10667698/23606891/fca9331a-0262-11e7-9b9e-2aeaca3a25cd.png">

Either use the indentation from 1148 or from the rest of the method. For new ones, the second one is preferred (4 spaces).